modin/core/dataframe/pandas/partitioning/axis_partition.py

```diff
@@ -595,10 +600,15 @@ def length(self):
 
     _width_cache = None
 
-    def width(self):
+    def width(self, materialize=True):
```

Comment:
We could pass `materialize` parameter to `self.list_of_block_partitions[0].width(materialize)`. The same for `def length`.